core/exception/security_error_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../../spec_helper'
+
+describe "SecurityError" do
+  # it "raises a SecurityError on a Regexp literal" do
+  #   -> { //.send(:initialize, "") }.should raise_error(SecurityError)
+  # end
+  begin
+    //.send(:initialize, "")
+  rescue SecurityError => exception
+    exception.message.should =~ /can't modify literal regexp/
+  end
+end
```

Comment:
New spec.

Reply:
That's hardly about SecurityError, much more about Regexp#initialize, could you move the spec there?
Use `raise_error(SecurityError) { |e| e.message.should =~ /can't modify literal regexp/ }`